SIL.Lift/Utilities.cs

```diff
@@ -63,11 +63,13 @@ static public string ProcessLiftForLaterMerging(string inputPath)
 			{
 				transform.Transform(outputOfPassOne, new XsltArgumentList(), output);
 			}
+			#if NET461
```

Comment:
Can you add a comment why this is not needed with netstandard 2?

Reply:
I had to add this, because in .NET Standard there is no .TemporaryFiles property. See the commit messages and the top of the PR page for more info!

Reply:
To be clear, I can't guarantee that the .NET Standard version doesn't use temporary files. I only know that that property isn't supported. I am definitely open to suggestions